swift-sdk/Constants.swift

```diff
@@ -55,6 +55,15 @@ public enum Const {
         static let payloadExpiration = 24
         static let attributionInfoExpiration = 24
     }
+    
+    enum PriorityLevel {
+        static let critical = 100.0
+        static let high = 200.0
+        static let medium = 300.0
+        static let low = 400.0
+        
+        static let unassigned = 300.5
```

Comment:
Why is this 300.5 and not `medium`?

Reply:
I realized I forgot to finalize the values of the levels and how to utilize them (e.g. `isCritical()`).

`300.5` is the true middle value since it also is right in the middle of the different possible campaign values.